includes/tracks/events/class-wc-extensions-tracking.php

```diff
@@ -21,6 +21,7 @@ public function init() {
 		add_action( 'woocommerce_helper_connected', array( $this, 'track_helper_connection_complete' ) );
 		add_action( 'woocommerce_helper_disconnected', array( $this, 'track_helper_disconnected' ) );
 		add_action( 'woocommerce_helper_subscriptions_refresh', array( $this, 'track_helper_subscriptions_refresh' ) );
+		add_action( 'wc_addons_woocommerce_payments_installed', array( $this, 'track_woocommerce_payments_install' ) );
```

Comment:
Not sure about hook name, it looks like all hooks above have `woocommerce_` prefix rather than `wc_`. Sine there were a request to have similar tracking for other extensions like WC Ship, would it make sense to have a shared action like `woocommerce_addon_installed` which would accept addon name as an argument.

Then it'd be called like `do_action( 'woocommerce_addon_installed', 'woocommerce_payments' );`

Reply:
Good call! I've made this change in https://github.com/woocommerce/woocommerce/pull/29052/commits/b999ae51839c8760023c5d8296e347155e62160b

Just noting that in order to follow the tracks naming conventions as closely as possible, I didn't programmatically create the event name using `$addon_id`:

> Don’t programmatically create event names. They should be hardcoded so they are easily searchable in the code base.

Reply:
> I didn't programmatically create the event name using $addon_id

💯 that's right approach.